test/labeling/test_analysis.py

```diff
@@ -105,7 +106,13 @@ def test_lf_summary(self) -> None:
 
         est_accs = [1, 0, 1, 1, 1, 0.5]
         names = list("abcdef")
-        df = self.LFA.lf_summary(self.Y, lf_names=names, est_accs=est_accs)
+
+        def f(x):
+            return -1
+
+        lfs = [LabelingFunction(s, f) for s in names]
+        lfa = LFAnalysis(np.array(L), lfs)
+        df = lfa.lf_summary(self.Y, est_accs=est_accs)
         df_expected = pd.DataFrame(
             {
                 "j": [0, 1, 2, 3, 4, 5],
```

Comment:
Should this be the names of the LFs (abcdef)? If not, is there a test to check that the names in the DataFrame are correct?

Reply:
yes this tests that the names are correct (see the index set below)